cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -286,6 +286,15 @@ class fixed_width_column_wrapper : public detail::column_wrapper {
       : fixed_width_column_wrapper(std::cbegin(elements), std::cend(elements)) {
   }
 
+  /**---------------------------------------------------------------------------*
+   * @brief Construct a non-nullable column of fixed-width elements from a vector
+   *
+   * @param element_list The vector of elements
+   *---------------------------------------------------------------------------**/
+  fixed_width_column_wrapper(std::vector<Element> elements)
```

Comment:
This shouldn't be necessary. It's trivial to emulate this with existing constructors:
```
fixed_width_column_wrapper<T> w(elements.begin(), elements.end());
```

Reply:
I added it to easily initialize `fixed_width_column_wrapper` using `{elements, bool_vec}`.
I removed it now.